lib/src/xdg_notification_portal.dart

```diff
@@ -59,19 +60,29 @@ class XdgNotificationActionInvokedEvent {
   /// Name of the action that was invoked.
   final String action;
 
-  const XdgNotificationActionInvokedEvent(this.id, this.action);
+  /// Additional information.
```

Comment:
```suggestion
  /// May contain additional information.
```

Since it isn't guaranteed to have any info at all.